presto-kafka/src/test/java/io/prestosql/plugin/kafka/schema/confluent/TestKafkaWithConfluentSchemaRegistryMinimalFunctionality.java

```diff
@@ -121,6 +121,22 @@ public void testTopicWithTopicRecordNameStrategy()
                         .build()));
     }
 
+    @Test
+    public void testUnsupportedInsert()
+    {
+        String topicName = "topic-unsupported-insert";
+        testingKafkaWithSchemaRegistry.createTopic(topicName);
+
+        assertNotExists(topicName);
+
+        List<ProducerRecord<Long, GenericRecord>> messages = createMessages(topicName, MESSAGE_COUNT, true);
+        sendMessages(messages, () -> testingKafkaWithSchemaRegistry.createConfluentProducer());
+
+        waitUntilTableExists(topicName);
```

Comment:
Do we need to push the data so as to access the table ?

Reply:
Yes, we need. Serializer is responsible to push schema to Schema Registry. If you don't write anything, then serializer won't serialize anything and schema won't appear in schema registry.